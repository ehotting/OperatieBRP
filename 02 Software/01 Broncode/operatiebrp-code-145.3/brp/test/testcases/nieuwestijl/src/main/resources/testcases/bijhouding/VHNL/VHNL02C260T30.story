Meta:
@auteur                 jozon
@status                 Klaar
@regels                 R1849
@sleutelwoorden         voltrekkingHuwelijkInNederland, VHNL02C260T30
@usecase                UCS-BY.HG

Narrative:
R1849 Aktenummer verplicht bij Nederlandse registerakten en leeg bij andere documentsoorten

Scenario: Aktenummer is niet gevuld bij Nederlandse akte
          LT: VHNL02C260T30



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/Libby.xls

When voer een bijhouding uit VHNL02C260T30.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Foutief
Then is het antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL02C260T30.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then is in de database de persoon met bsn 422531881 niet als PARTNER betrokken bij een HUWELIJK

Then lees persoon met anummer 1868196961 uit database en vergelijk met expected VHNL02C260T30-persoon1.xml
