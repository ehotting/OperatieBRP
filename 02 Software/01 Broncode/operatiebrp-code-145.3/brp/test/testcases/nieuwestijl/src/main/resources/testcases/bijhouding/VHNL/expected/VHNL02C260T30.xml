<?xml version="1.0" encoding="UTF-8"?><brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">&#13;
   <brp:stuurgegevens>&#13;
      <brp:zendendePartij>199901</brp:zendendePartij>&#13;
      <brp:zendendeSysteem>BRP</brp:zendendeSysteem>&#13;
      <brp:referentienummer>*</brp:referentienummer>&#13;
      <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>&#13;
      <brp:tijdstipVerzending>*</brp:tijdstipVerzending>&#13;
   </brp:stuurgegevens>&#13;
   <brp:resultaat>&#13;
      <brp:verwerking>Foutief</brp:verwerking>&#13;
      <brp:hoogsteMeldingsniveau>Deblokkeerbaar</brp:hoogsteMeldingsniveau>&#13;
   </brp:resultaat>&#13;
   <brp:meldingen>&#13;
      <brp:melding brp:objecttype="Melding" brp:referentieID="comid.voltrekkingHuwelijkInNederland.registratieAanvangHGP.bron1">&#13;
         <brp:regelCode>R1606</brp:regelCode>&#13;
         <brp:soortNaam>Deblokkeerbaar</brp:soortNaam>&#13;
         <brp:melding>*</brp:melding>&#13;
      </brp:melding>&#13;
   </brp:meldingen>&#13;
   <brp:voltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">&#13;
      <brp:partijCode>507013</brp:partijCode>&#13;
      <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>&#13;
   </brp:voltrekkingHuwelijkInNederland>&#13;
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
